lib/corefoundation.rb

```diff
@@ -12,6 +12,7 @@
 require_relative "corefoundation/date"
 
 require_relative "corefoundation/error"
```

Comment:
is this still being used?

Reply:
This was not in use anywhere. Removal falls under the update ruby gem PR and not this. Once that is updated after I get back to it, this PR will be rebased and it will be gone. Making the switch to an approach based off of Exceptions module in infra client.